src/js/themes/base.d.ts

```diff
@@ -1039,8 +1039,12 @@ export interface ThemeType {
   };
   notification?: {
     container?: BoxProps;
+    direction?: 'column' | 'row';
+    truncate?: boolean;
     toast?: {
```

Comment:
I think we should support the background color for toast working similar to how direction and truncate work. As in, if not defined for toast they will use what the base notification has under the status keys. So, `theme.notification.toast.critical.background` would take precedence for toast before `theme.notification.critial.background` but if it isn't defined under `toast`, then the second one would be applied. I realize this means explicitly turning off the background for the HPE theme, but I think it mirrors general grommet patterns better.

Reply:
That's fair, let me look at making those adjustments.